src/views/forms/TradeForm/PlaceOrderButtonAndReceipt.tsx

```diff
@@ -121,6 +119,13 @@ export const PlaceOrderButtonAndReceipt = ({
     },
   ];
 
+  const returnToMarketState = {
```

Comment:
[q] will we ever interact with this value?
for example:
```
buttonStatesPerStep[MobilePlaceOrderSteps.PlacingOrder].buttonAction = someNewAction
```
if so then may be worth making this a function instead. otherwise no response needed!